### DISCUSSION TITLE
Como Vincular el correo de innovaccion virtual a tu cuenta ya existente de GitHub

### DISCUSSION THREAD

**Discussion by OscarTinajero117** [Launch-X-Latam/MisionFrontEnd#28] [Show and tell]
Les cuento, si ya tienen su cuenta en uso y no desean tener otra cuenta (esto para centralizar todo, lo cual es correcto a mi parecer), hagan lo siguiente:

**Dar clic aquí**
![image](https://user-images.githubusercontent.com/79131873/192412062-9ba85558-33ea-4fed-8f57-5b881ba6974d.png)

**Después en 'Settings' (Configuración, en caso de que esten traduciendo el sitio)**
![image](https://user-images.githubusercontent.com/79131873/192412222-265753b9-7ad9-4657-99fa-9c2253a8c090.png)

**En la sección de 'Access' dan clic en 'Emails'**
![image](https://user-images.githubusercontent.com/79131873/192412362-70566262-5665-4f87-90ee-bb285472df40.png)

**En Emails pueden ver sus emails agregados, agregar nuevos emails y seleccionar el email principal**
![image](https://user-images.githubusercontent.com/79131873/192412769-500cafcb-bf0a-4152-9319-f642af71ecad.png)

**Comment by DaviiidSG** [comment]
Disculpa y qué ventajas trae hacer eso?? es que no sé la utilidad jajajaja

**Reply by OscarTinajero117** [reply]
Centralizar tu protafolio, te sirve para cuando un reclutador vea tu perfil de GitHub, vea lo que haz hecho, tu actividad, etc.

**Comment by magouwuxd** [comment]
oye, y si cree otra cuenta con el correo de innovaccion y mi github esta separado que se hace ahi?

**Reply by OscarTinajero117** [reply]
Checa este [link](https://docs.github.com/es/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts)

Espero te sirva

**Comment by rgplazas** [comment]
Gracias
